xdsl/dialects/arm/__init__.py

```diff
@@ -0,0 +1,16 @@
+from xdsl.ir import Dialect
+
+from .register import GeneralRegisterType
+
+"""
+ARM dialect, based on the ISA specification in:
+https://developer.arm.com/documentation/102374/0101/Overview
+"""
```

Comment:
A string is recognised as a doc string if it's the first thing in the file, can you please move this up?

Reply:
thanks for catching that!

Reply:
This is one of the reasons to let the original commenters resolve the comments :)